dev_docs/01_getting_started.md

```diff
@@ -55,3 +55,4 @@ The guidelines referenced above should be sufficient for the most common tasks.
 - [How to update the documentation website](./04_how_to_update_docs.md)
 - [Icons](./05_icons.md)
 - [Miscellaneous](./06_misc.md)
+- [Visual Testing](./07_visual_testing_guide.md)
```

Comment:
I think we could even move this to the first item in the list, as presumably it will be the most frequently navigated to, compared to others :)